assets/javascripts/views/content/settings_page.coffee

```diff
@@ -17,23 +14,18 @@ class app.views.SettingsPage extends app.View
     settings.dark = app.settings.get('dark')
     settings.smoothScroll = !app.settings.get('fastScroll')
     settings.arrowScroll = app.settings.get('arrowScroll')
-    settings[layout] = app.settings.hasLayout(layout) for layout in LAYOUTS
+    settings[layout] = app.settings.hasLayout(layout) for layout in app.settings.getAllLayouts()
```

Comment:
Why not refer to the `LAYOUTS` variable directly?

```suggestion
    settings[layout] = app.settings.hasLayout(layout) for layout in app.settings.LAYOUTS
```

This requires a small change to the definition:

```coffee
LAYOUTS: ['_max-width', '_sidebar-hidden', '_native-scrollbars']
```

and the single use in the `Settings` class:

```coffee
@toggleLayout(layout, @hasLayout(layout)) for layout in @LAYOUTS

Reply:
Thanks, fixed!